critiquebrainz/db/avg_rating_test.py

```diff
@@ -0,0 +1,103 @@
+from critiquebrainz.data.testing import DataTestCase
+import critiquebrainz.db.users as db_users
+from critiquebrainz.db.user import User
+import critiquebrainz.db.review as db_review
+import critiquebrainz.db.avg_rating as db_avg_rating
+import critiquebrainz.db.exceptions as db_exceptions
+import critiquebrainz.db.license as db_license
+
+class AvgRatingTestCase(DataTestCase):
+
+    def setUp(self):
+        super(AvgRatingTestCase, self).setUp()
+
+        self.user = User(db_users.get_or_create("Tester", new_user_data={
+            "display_name": "test user",
+        }))
+        self.user_2 = User(db_users.get_or_create("Tester 2", new_user_data={
+            "display_name": "test user 2",
+        }))
+        self.license = db_license.create(
+            id=u'Test',
+            full_name=u"Test License",
+        )
+
+    def test_insert_and_get(self):
+        """Test if rating and count are inserted on creation of the first review with rating
+        and the get function that returns a dict containing rating and count"""
+
+        review = db_review.create(
+            entity_id="e7aad618-fa86-3983-9e77-405e21796eca",
+            entity_type="release_group",
+            text=u"Testing!",
+            rating=100,
+            user_id=self.user.id,
+            is_draft=False,
+            license_id=self.license["id"],
+        )
+        avg_rating = db_avg_rating.get(review["entity_id"], review["entity_type"])
+        self.assertEqual(avg_rating["entity_id"], review["entity_id"])
+        self.assertEqual(avg_rating["entity_type"], review["entity_type"])
+        self.assertEqual(avg_rating["rating"], review["rating"])
+        self.assertEqual(avg_rating["count"], 1)
```

Comment:
Instead of having multiple statements you can just compare one dictionary to another once.